changelog/druntime.importc-initial-msvc-intrinsics.dd

```diff
@@ -0,0 +1,4 @@
+ImportC, when targeting the Microsoft C runtime, supports a subset of the intrinsics recognised by the MSVC compiler.
```

Comment:
Should American English spellings be used for changelog entries? ("recognised")

Reply:
yes

Reply:
Noted. (That might be worth documenting in https://github.com/dlang/dmd/blob/75dca527f98b806fd82427d839f76df8290a1b00/changelog/README.md)